cpp/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+
+set(SOURCE_FILES
+        EncryptionProvider.cpp
+        FileTransferAgent.cpp
+        SnowflakeS3Client.cpp
+        StorageClientFactory.cpp
+        )
+
+add_library(snowflakeclientcpp STATIC ${SOURCE_FILES})
```

Comment:
Is it possible to merge both C and C++ into one library? e.g., libsnowflakeclient.a

Reply:
Sure, I will make it one library.